packages/lit-dev-content/site/docs/v3/releases/upgrade.md

```diff
@@ -1,40 +1,57 @@
 ---
-title: Upgrade guide
+title: Lit 3 upgrade guide
 eleventyNavigation:
-  key: Upgrade guide
+  key: Lit 3 upgrade guide
   parent: Releases
   order: 3
 versionLinks:
   v2: releases/upgrade/
 ---
 
-## Overview
+<div class="alert alert-info">
+
+If you are looking to migrate from Lit 1.x to Lit 2.x, see the [Lit 2 upgrade guide](/docs/v2/releases/upgrade/).
+
+</div>
 
-Lit 3.0 has very few breaking changes from Lit 2.0. Any projects using Lit 2
-without deprecation warnings should not require any changes to upgrade to Lit 3.
+## Overview
 
-Lit 3.0:
+Lit 3.0 has very few breaking changes from Lit 2.x:
 
-- Drops support for IE11
-- Is published as ES2021
-- Removes a few deprecated APIs
+- IE11 is no longer supported.
+- Lit's npm modules are now published as ES2021.
+- APIs marked deprecated in Lit 2.x releases have been removed.
+- SSR hydration support modules have moved to the `@lit-labs/ssr-client` package.
+- Decorator behavior has been unified between TypeScript experimental decorators and standard decorators.
+- Support was removed for Babel decorators version "2018-09"
 
-Lit 3.0 should require no changes to upgrade from Lit 2.0 for the vast majority
+Lit 3.0 should require no changes to upgrade from Lit 2.x for the vast majority
 of users. Most apps and libraries should be able to extend their npm version
 ranges to include both 2.x and 3.x, like `"^2.7.0 || ^3.0.0"`.
 
-Lit 2.x and 3.0 are _interoperable_: templates, base classes, directives,
-decorators, etc., from one version of Lit will work with those from another.
+Lit 2.x and 3.0 are _interoperable_: templates, base classes, and directives from one version of Lit will work with those from another.
+
+## Updates to Lit decorators
+
+The Lit 3 decorators are largely backwards compatible. If you've been using them with TypeScript, **most likely no changes are needed**. We've added forward compatibility with standard decorators, but the legacy `experimentalDecorators: true` without the `accessor` keyword is currently the more performant option.
```

Comment:
I think this could use a bit more context setting, even though we'll repeat this somewhat elsewhere:

```suggestion

JavaScript decorators have recently been standardized by TC39 and are at Stage 3 of the four-stage standardization process. Stage 3 is when JavaScript implementations such as VMs and compilers start implementing the stable specification. TypeScript 5.2 and Babel 7.23 have implemented the standard recently.

This means that there is more than one version of the decorator API in existence: standard decorators, TypeScript's experimental decorators, and previous proposals that Babel has implemented, such as version "2018-09".

While Lit 2 supported TypeScript experimental decorators and Babel's "2018-09" decorators, Lit 3 now supports standard decorators and TypeScript experimental decorators.

The Lit 3 decorators are mostly backwards compatible with the Lit 2 TypeScript decorators - **most likely no changes are needed**.
```